crates/floresta-chain/src/pruned_utreexo/chain_state_builder.rs

```diff
@@ -169,14 +170,24 @@ impl<T: ChainStore> ChainStateBuilder<T> {
     }
 
     /// Get the specified best tip as a `BestChain`, or fall back to the genesis block if unset.
-    /// Returns an error if chain parameters are missing when determining the genesis block.
+    /// The returned validation index is always the genesis block (to skip validation, you must
+    /// specify an `assume_valid` hash).
```

Comment:
To fully skip you need to use `assume_utreexo`. `assume_valid` only skips script validation